inspirehep/modules/theme/static/scss/feedback/modal.scss

```diff
@@ -18,6 +18,53 @@
  * 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
  */
 
+.modal-header {
+  background-color: $dark-blue;
+  color: white;
+  .close {
+      color: white; 
+      opacity: 1;
+  }
+}
+
+.modal-dialog {
+  width: 60%;
+  margin: 0px;
+  position: fixed;
+    top: 30%;
```

Comment:
Indentation is off here, but that's ok: the real issue to fix here is adding some CSS linting step to our build. Perhaps we can postpone this to the general rearchitecture of the UI code.